src/utils/valid.js

```diff
@@ -1,50 +1,32 @@
-export function validEmail(target) {
-  if (target.validity.valueMissing) return "이메일을 입력해주세요";
-  else if (target.validity.typeMismatch) return "잘못된 이메일 형식입니다";
-  return "";
-}
+const regaxEmail =
+  /^[0-9a-zA-Z]([-_\\.]?[0-9a-zA-Z])*@[0-9a-zA-Z]([-_\\.]?[0-9a-zA-Z])*\.[a-zA-Z]{2,3}$/i;
 
-export function validNickname(target) {
-  if (target.validity.valueMissing) return "닉네임을 입력해주세요";
+export function validEmail(value) {
+  if (value === "") return "이메일을 입력해주세요";
+  else if (!regaxEmail.test(value)) return "잘못된 이메일 형식입니다";
   return "";
 }
 
-export function validPassword(target) {
-  if (target.validity.valueMissing) return "비밀번호를 입력해주세요";
-  else if (target.validity.tooShort) return "비밀번호를 8자 이상 입력해주세요";
+export function validNickname(value) {
+  if (value === "") return "닉네임을 입력해주세요";
   return "";
 }
 
-export function validConfirmPassword(password, confirmPassword) {
-  if (!confirmPassword) {
-    return;
-  } else if (password.value === confirmPassword?.value) {
-    confirmPassword.nextSibling?.remove();
-    confirmPassword.classList.remove("auth__invalid");
-  } else {
-    displayInvalidMessage(confirmPassword, "비밀번호가 일치하지 않습니다..");
-  }
+export function validPassword(value) {
+  if (value === "") return "비밀번호를 입력해주세요";
+  else if (value.length < 8) return "비밀번호를 8자 이상 입력해주세요";
   return "";
 }
 
-function displayInvalidMessage(target, message) {
-  let span;
-  if (target.nextSibling?.tagName === "SPAN") {
-    span = target.nextSibling;
-  } else {
-    span = document.createElement("span");
-    target.after(span);
-  }
-  span.style.padding = "8px 16px 0px";
-  span.textContent = message;
-  span.style.color = "#f74747";
-  target.classList.add("auth__invalid");
+export function validPasswordConfirm(value1, value2) {
```

Comment:
```suggestion
export function validPasswordConfirm(value1, value2) {
```
value 1, value 2 는 좋은 이름이 아닙니다.

어떤 값이 들어올지 예상 할 수 있는 이름으로 지어주세요~